E/Eigen/build_tarballs.jl

```diff
@@ -0,0 +1,43 @@
+using BinaryBuilder
+
+name = "Eigen"
+version = v"3.3.7"
+
+sources = [
+    ArchiveSource("https://bitbucket.org/eigen/eigen/get/$(version).tar.bz2",
+                  "9f13cf90dedbe3e52a19f43000d71fdf72e986beb9a5436dddcd61ff9d77a3ce")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/eigen-eigen-323c052e1731
+
+mkdir build
+cd build/
+
+cmake -DCMAKE_INSTALL_PREFIX=$prefix ..
+make -j${nproc}
+make install
+cd ..
+cp cmake/FindEigen3.cmake ${prefix}/share/eigen3/cmake/
+for lic in $(ls COPYING.*)  # different licences apply to different parts of Eigen
+do
+  install_license ${lic}
+done
```

Comment:
`install_license` does the loop already for you :slightly_smiling_face: 
```suggestion
install_license COPYING.*
```
Also, never use `ls` in this way: https://mywiki.wooledge.org/ParsingLs.  You could have just done
```sh
for lic in COPYING.*
```
:wink: